sdk/openai/openai/sources/customizations/api/policies/formDataPolicy.ts

```diff
@@ -65,7 +65,10 @@ function prepareFormData(formData: FormDataMap, request: PipelineRequest): void
     }
   }
 
-  const encoder = new FormDataEncoder(requestForm);
+  // This library doesn't define `type` entries in the exports section of its package.json.
+  // See https://github.com/microsoft/TypeScript/issues/52363
+  const { FormDataEncoder } = await import("form-data-encoder" as any);
```

Comment:
Why is there an as any inside the await import statement?

Reply:
I tried to explain it in the comment right above, their "exports" section doesn't define types for the exported modules.